.goreleaser.yml

```diff
@@ -115,11 +115,16 @@ snapcrafts:
     grade: stable
     confinement: strict
     publish: true
+    plugs:
+      personal-files:
+        read:
+        - $HOME/.ttn-lw-stack.yml
+        - $HOME/.ttn-lw-cli.yml
```

Comment:
Currently we provide 3 different ways to configure the configuration file: https://github.com/TheThingsNetwork/lorawan-stack/blob/19b035b63739d3c0f562927b07a1475bf36d13c8/doc/content/guides/getting-started/cli/_index.md#L37-L39
This only supports one of them in a very specific location.

1. Is it possible to support all 3 configuration options (i.e. env variable pointing to arbitrary location, cli flag pointing to arbitrary location, config in `XDG_CONFIG_HOME` or working directory). Maybe there's a way the user can configure the allowed paths locally?
2. If (1.) is not possible, then we should at least support the other 2 possible locations as well - in `XDG_CONFIG_HOME` and in working directory.

Reply:
The issue only happens for `$HOME/.ttn-lw-cli.yml`, permissions for the working directory and XDG_CONFIG_HOME are OK as-is.